### ISSUE TITLE
Question about training loss

### ISSUE THREAD

**Issue by gray311** [Reason-Wang/flan-alpaca-lora#5]
Hi, I'm very interested in your project, but during the training, I found that the training loss will be very big, more than 30, is it normal?

**Comment by Reason-Wang** [comment]
I don't think this is normal. Could you provide more details for training (model, data, command, and package versions)?
